var/spack/repos/builtin/packages/py-region-grower/package.py

```diff
@@ -21,8 +22,8 @@ class PyRegionGrower(PythonPackage):
 
     depends_on('py-click@7.0:', type='run')
     depends_on('py-tqdm@4.28.1:', type='run')
-    depends_on('py-tns@2.2.5:', type='run')
+    depends_on('py-tns@2.4.2:', type='run')
     depends_on('py-voxcell@2.6.3:', type='run')
-    depends_on('py-diameter-synthesis@0.1.7:', type='run')
-    depends_on('py-morphio@2.3.4:', type='run')
+    depends_on('py-diameter-synthesis@0.2.3:', type='run')
+    depends_on('py-morphio@3.1.1:', type='run')
```

Comment:
Some dependencies are different from `setup.py`, that in `0.2.1` defines for `install_requires`:
```
    "attrs>=19.3.0",
    "click>=7.0",
    "dask[dataframe, distributed]>=2.15.0",
    "diameter-synthesis>=0.2.3",
    "morphio>=2.7",
    "morph-tool[nrn]>=2.4.1",
    "neuroc>=0.2.3",
    "neurom>=2,<3",
    "tns>=2.4.0",
    "tqdm>=4.28.1",
    "voxcell>=2.7",
```